server/controllers/group_controller.js

```diff
@@ -1,37 +1,37 @@
 import Group from '../models/group';
 
-const ObjectId = require('mongoose').Types.ObjectId;
-
-export const createGroup = (name, description, creator) => {
-  // TODO: error check on names, descriptions and creators?
+export const createGroup = (name, description, userId) => {
   const group = new Group();
   group.name = name;
   group.description = description;
-  group.creator = creator; // TODO: @ploomis how to get authenticated user ??
+  group.creator = userId; // how to get authenticated user ??
+
   group.createDate = Date.now();
   group.editDate = Date.now();
-  group.members.push(ObjectId(creator));   // TODO: make sure creator is a valid user ID
+  group.members.push(userId);   // TODO: make sure creator is a valid user ID
 
   return group.save(); // TODO: catch errors
 };
 
-export const addGroupMember = (groupid, memberid) => {
+export const addGroupMember = (groupIds, userId) => {
 // TODO: similar to addGroupArticle
-  Group.findOne({ _id: ObjectId(groupid) }, (err, group) => {
-    group.members.push(memberid); // TODO: validate member is a real member
-    return group.save();
-  });
+  for (let i = 0; i < groupIds.length; i++) {
+    const groupId = groupIds[i];
+    Group.findByIdAndUpdate(groupId, { $push: { members: userId } });
```

Comment:
For these, I don't think I can return within the for-loop. I'll look more into a better way to do bulk updates.